app/views/directives/mobile-service-clients.html

```diff
@@ -0,0 +1,33 @@
+<div class="row mobile-clients">
+  <div class="col-sm-6">
+    <div class="client-title component-label section-label">Mobile Clients</div>
+    <div ng-repeat="mobileClient in $ctrl.filteredClients" class="col-sm-12 col-md-6 mobile-client">
+      <span class="fa-2x {{mobileClient.metadata.annotations.icon}} client-icon"></span>
+      <div class="client-details">
+        <a ng-href="{{mobileClient | navigateResourceURL}}">{{mobileClient.spec.name}}</a>
+        <div class="client-type">{{mobileClient.spec.appIdentifier}}</div>
+      </div>
+      <div class="exclude" ng-click="$ctrl.excludeClient(mobileClient)">
+        <span class="pficon-error-circle-o"></span>
+        <span>Exclude Client</span>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-xs-12 col-sm-6">
+    <button ng-disabled="$ctrl.canAddMobileClient()" class="btn btn-default btn-lg pull-right"
+            ng-click="$ctrl.showOverlayPanel('addMobileClient', {target: $ctrl.serviceInstance})">Add Client
+    </button>
+  </div>
+
+  <overlay-panel show-panel="$ctrl.overlay.panelVisible" handle-close="$ctrl.closeOverlayPanel">
+    <div ng-if="$ctrl.overlay.panelName === 'addMobileClient'">
```

Comment:
Instead of `$ctrl.overlay.panelName === 'addMobileClient'` filter `addMobileClient` can be created and you can pass it like `$ctrl.overlay.panelName | addMobileClient`

Reply:
@vchepeli we have followed pattern set out in the upstream https://github.com/sedroche/origin-web-console/blob/f24bf3ae2d51e3912845fbdd7ea8c24dda9389cd/app/views/overview/_service-instance-row.html#L192